python/orca/example/learn/pytorch/super_resolution/README.md

```diff
@@ -44,7 +46,7 @@ python super_resolution.py --cluster_mode local
 python super_resolution.py --cluster_mode yarn
 ```
 
-You can run this example with bigdl backend (default) or torch_distributed backend. 
+You can run this example with bigdl backend (default) or torch_distributed backend.

```

Comment:
also need to modify this sentence?

Reply:
deleted a space.

Reply:
I mean need to add spark backend here in the description?